docs/arch/adr-24.md

```diff
@@ -0,0 +1,64 @@
+
+- RHC: A right handed coordinate system, this is defined as a system where positive angles rotate an object from the +x axis to the +y axis with angle=0 along the +x axis
+
+- LHC: A left handed coordinate system, this is defined as a system where positive angles rotate an object from the +x axis to the -y axis with angle=0 along the +x axis
+
+- Domain: The range of values a number can take [ or ] is inclusive, ( or ) is exclusive.
+
+- Upper origin: The point 0, 0 is in the upper left of the plot
+
+- Lower origin: The point 0, 0 is in the lower left of the plot
+
+The inconsistencies comprise of:
+
+- The browser SVG coordinate systems use _RHC_ with an _upper origin_ resulting in positive angles rotating clockwise. Most plotting software (R, Python, Matlab) are _RHC_ with a _lower origin_ resulting in positive angles rotating counter-clockwise. 
+  - The position of origin has been inconsistent between tools which has an effect on the final annotation too. Most use the center x, y point, but some don't
+  - Some of the drawing tools use _LHC_
+- Some tools' annotation use `angle` some use `rotation`
+- It's unclear when the x, y annotation refers to the center point of the shape
+- It's unclear when the x, y annotation is being used as the point of rotation
+
+Some of the mark annotation models have a few other issues as well:
+
+- Some shapes have default values which an create bias. For example, the ellipse has a default axis ratio of 0.5 and many volunteers have left the default creating a bias ([comment](https://github.com/zooniverse/front-end-monorepo/issues/500#issuecomment-516788821))
+- The freehand drawing tool has peformance impact on the browser as the drawing is being created and with the job to create classification exports as well. This is because the current annotation consists of every single x, y point created
+
+## Decision
+
+The shape's mark annotation models should change for consistency and improved post-classification analysis in the following ways:
+
+- The annotation should use the mathematical standard of _RHC_ with a domain of `(-180, 180]` for consistent angle calculation
+- The annotation model should use `angle` for naming rotation angles. This replaces usage of `rotation`.
+- The annotation model should replace `x` and `y` with `x_center` and `y_center` for shapes where that is applicable
+  - The exceptions are non-shapes like point, line, and transcription line tools, and non-symmetric shapes like fan.
+  - Shape clustering in aggregation is always done with the center
+- All rotations should be defined about `x_center` and `y_center` point. If the rotation cannot be defined around the center point, then the point used should be clearly recorded in the annotation as `x_rotation` and `y_rotation`
```

Comment:
Having `{ x, y }`, `{ x_center, y_center }` and `{ x_rotation, y_rotation }` still makes me uneasy in terms of how we'll fit that into the SVG rendering code.

Reply:
Line 43 suggesting we leverage MST's computed views addresses this concern. I'll add an example as asked.

Reply:
I've updated to include a code example. Does this address your concern? If not, could you be specific about what your concerns are?

Reply:
It does. I don't see why we have `{ x, y }`, { x_center, y_center }` and `{ x_rotation, y_rotation }` if they are mutually exclusive. It only makes the models clumsy and harder to maintain.

Reply:
For downstream analysis. @CKrawczyk asked for explicit naming for what the point represents. I don't agree that it is clumsy as developers will have to think specifically about the shape of the tool (is it symmetrical for instance) when building a new one as well.

Reply:
Also to be clear, we won't have a `{ x, y }` in any of the shapes anymore. They almost all will be replaced with `{ x_center, y_center }` with the exception of shapes that have a point of rotation elsewhere like the fan tool.

Reply:
Line 39 mentions the replacement in the models.

Reply:
I could very well be wrong but I read @CKrawczyk's comments as asking for `{ x_center, y_center }` in addition to `{ x, y }`, not in place of. Likewise, recording the centre of rotation _and_ the centre of the shape when they're different.

Reply:
The original request in #500 included: 

> x and y should be consistently defined as the center of a shape (with exception of the fan tool as this is not symmetric). All shape clustering needs to be done with the center anyways, so this saves a set when analyzing the data.
> x and y could be renamed x_center and y_center to make this clear to the science teams

@CKrawczyk could you confirm that you would like `{ x, y }` to be renamed as `{ x_center, y_center }` where applicable? For the fan tool, which is the only non-symmetrical shape we have I think, do you want `{ x, y  }` to be replaced by `{ x_rotation, y_rotation }` and have the center be included separately, too? If so, would that be true for other non-symmetrical shapes?

Reply:
I think the only shape that affects is the rectangle, where we'd change `{ x, y }` to record the centre. I think it's currently recording the top left corner, which isn't very useful. If we always use `{ x, y }` for clustering then it makes sense to me to name them consistently, however we name them. What I'd like to avoid is each shape having its own rules for naming the same thing (ie. the point used for clustering.)

Reply:
Because non-symmetrical shapes don't use the center of the shape as the point of rotation, I think @CKrawczyk is suggesting we use a naming convention that clearly distinguishes this. This does introduce some logic to manage in downstream, but it sounds like knowing absolutely what the point represents is more important?